numpy/lib/tests/test_format.py

```diff
@@ -927,6 +929,15 @@ def test_empty_npz():
     np.savez(fname)
     np.load(fname)
 
+def test_loading_invalid_zip_file():
+    with zipfile.ZipFile('spam.zip', 'w') as myzip:
+        myzip.writestr('spam.zip', 'eggs')
+    # Corrupting spam.zip intentionally
+    with open('spam.zip', 'w+b') as fh:
+        fh.seek(2)
+        fh.write(b"\xde\xad\xc0\xde")
+    assert_raises(zipfile.BadZipFile, NpzFile, 'spam.zip')
```

Comment:
I don't know if this test is interesting, it would pass before this patch too.

Reply:
I agree - the goal would be to test the ignored exception (i.e. the `AttributeError` that is raised as a side-effect) is *not* raised with an invalid zip file. I'm not sure how to design such a test, but this test should not be included as is IMO.